event_test.go

```diff
@@ -0,0 +1,67 @@
+package xendit_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/xendit/xendit-go"
+)
+
+func TestEventInvoicePaid_EWallet(t *testing.T) {
```

Comment:
Please help to rename this test to reflect the new callback struct, thank you!

Reply:
thanks ! 
i also add test for bank, retail outlet, and expired cases. 
copy and pasted the test payload from the [docs](https://developers.xendit.co/api-reference/#invoice-callback)